### PR TITLE
New/home

### PR DESCRIPTION
**Pull request by macjank**

### CODE REVIEW

File: src/components/Home/HomeGame.tsx [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+import { Link } from 'react-router-dom';
+
+type HomeGameProps = {
+  id: string;
+  title: string;
+  src: string;
+};
+
+const HomeGame = ({ id, title, src }: HomeGameProps) => {
+  return (
```
Comment (kubaczerwinski77) [inline src/components/Home/HomeGame.tsx:11]: I would suggest that styling
```
<Box
      component={Link}
      to={`/games/${id}`}
      sx={{
        backgroundColor: '#1F1F1FBB',
        borderRadius: '5px',
      }}
    >
      <Image radius={3} p={5} src={src} alt={title} />
</Box>
```

File: src/App.tsx [hunk 2]
```diff
@@ -1,11 +1,23 @@
 import { QueryClient, QueryClientProvider } from 'react-query';
 import Shell from './components/shell/Shell';
+import { Global } from '@mantine/core';
+import background from './assets/rose-petals_1.svg';
```
Comment (pastepi) [inline src/App.tsx:7]: Font is still default, we decided to use Poppins, you can check how I managed to implement it [here](https://github.com/CodersCamp2021/team-lp-project-4/blob/new/games-view/src/App.tsx) (don't forget to install the font from [fontsource](https://fontsource.org/)).

File: src/components/Home/Home.tsx [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+
+const Home = () => {
+  const { classes } = useStyles();
+  return (
+    <Container className={classes.homeView}>
+      <Stack className={classes.textContainer}>
+        <Text className={classes.title} color="#ffffff">
+          Let them play your game.
+        </Text>
+        <Text className={classes.subtitle} color="#cccccc">
```
Comment (kubaczerwinski77) [inline src/components/Home/Home.tsx]: Changed subtitle to look more like in the Figma project
```
<Text
          className={classes.subtitle}
          color="#cccccc"
          transform="uppercase"
          sx={{ fontWeight: 'bold', letterSpacing: '6px' }}
        >
          Discover, review,
          <br /> enjoy yourself
</Text>
```

**Review by kubaczerwinski77 — CHANGES_REQUESTED** [review]
Just two small style-like changes, the rest - great 💯

**Review by pastepi — CHANGES_REQUESTED** [review]

**Review by pastepi — APPROVED** [review]

**Pull request merged**